plugins/http/plugin.go

```diff
@@ -15,14 +15,14 @@ import (
 	"github.com/hashicorp/go-multierror"
 	"github.com/spiral/endure"
 	"github.com/spiral/errors"
+	config2 "github.com/spiral/roadrunner/v2/interfaces/config"
```

Comment:
why alias it?

Reply:
Nice catch. Actually, this alias is the result of file move auto-refactoring and at the time replaced with correct.